lib/ansible/module_utils/urls.py

```diff
@@ -1383,7 +1383,7 @@ def fetch_url(module, url, data=None, headers=None, method=None,
         info.update(dict(msg="OK (%s bytes)" % r.headers.get('Content-Length', 'unknown'), url=r.geturl(), status=r.code))
     except NoSSLError as e:
         distribution = get_distribution()
-        if distribution is not None and distribution.lower() == 'redhat':
+        if distribution is not None and distribution.lower() == 'rhel':
```

Comment:
This seems to indicate that there may be issues with anyone else using `get_distribution` outside of us.  I feel like we might need to normalize this to match previous values.

Reply:
`get_distribution()` now returns `Redhat` for RHEL systems.